_posts/2019-07-23-writing-tizen-net-applications-in-fsharp.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Writing Tizen .NET Applications in F#"
+last_modified_at: 2019-07-23
+categories: 
+  - Tizen.NET
```

Comment:
`Tizen .NET`
there should be a blank between words.

Reply:
Fixed.